src/functions/uploadToRSIS/application/data-mapper/cat-adi2-mapper.ts

```diff
@@ -0,0 +1,377 @@
+import { ResultUpload } from '../result-client';
+import { DataField } from '../../domain/mi-export-data';
+import {
+  field,
+  formatManoeuvreFault,
+  formatMultipleManoeuvreFaults,
+  formatManoeuvreSerious,
+  optional,
+  optionalBoolean,
+  formatManoeuvreDangerous,
+  addIfSet,
+  getCompetencyComments,
+  formatQuestionFaultADI2,
+  formatQuestionCompleted,
+} from './data-mapper';
+import { formatGearboxCategory } from '../helpers/shared-formatters';
+import { CatADI2UniqueTypes } from '@dvsa/mes-test-schema/categories/ADI2';
+import TestData = CatADI2UniqueTypes.TestData;
```

Comment:
same as above in reference to not the typical way of importing TestData

Reply:
TestData isn't exported directly, so can only reference it via CatADI2UniqueTypes.

Reply:
is it not this?

import { TestData } from '@dvsa/mes-test-schema/categories/ADI2/partial';

Reply:
removed import of test data and am accessing where needed directly from CatADI2UniqueTypes